### PR TITLE
6987 Customize Tailwind Spacing Units

### PR DESCRIPTION
**Pull request by fessehaye**
Redoing spacing units based on Figma design system, should be reviewed before people try to use tailwind for other tickets not related to the Tailwind Epic!

Also rebuilt the dev-environment for reloading HTML/tailwind changes.

On a separate terminal from the docker one, run `npm run tailwind` and trying changing the html files and their classes.
Changes should appear on `localhost:3000`

Old Default Spacing:
https://tailwindcss.com/docs/customizing-spacing

New Spacing: 
```
    spacing: {
      0: "0",
      px: "1px",
      1: "0.25rem",
      2: "0.5rem",
      3: "0.75rem",
      4: "1rem",
      5: "1.5rem",
      6: "2rem",
      7: "3rem",
    },
```

### CODE REVIEW

**Review by KalobTaulien — CHANGES_REQUESTED** [review]
As far as I can tell the code looks good. But percy has flagged some margin/padding changes. Let's get a designer to review this as well.

**Review by sabrinang — DISMISSED** [review]
For the figma design file, I've added the sizing/spacers on the [grid & layout page](https://www.figma.com/file/JT9Dvdyc96AqQ3HJlnA9Qq/?node-id=312%3A354) and in the plug-in: 
image

@fessehaye I was able to add 0, px, 2.5 as figma components however I can't add "2.5" as a design token in [the plugin](https://docs.tokens.studio/tokens/creating-tokens) as the . separates the name and doesn't store the token in the plugin. Any tips on how you are naming fraction values and I don't know if you had the same issue in the code?

File: network-api/networkapi/context_processor.py [hunk 1]
```diff
@@ -16,3 +16,7 @@ def canonical_path(request):
 
 def canonical_site_url(request):
     return {'CANONICAL_SITE_URL': request.scheme + '://' + request.get_host()}
+
+def env_debug(request):
+    debug_flag = settings.DEBUG
+    return{"debug_flag":debug_flag}
```
Comment (KalobTaulien) [inline network-api/networkapi/context_processor.py:22]:
```suggestion
    return {"debug": settings.DEBUG}
```

File: network-api/networkapi/templates/pages/base.html [hunk 2]
```diff
@@ -31,7 +31,9 @@
     {% block commento_meta %}{% endblock %}
 
     {% block stylesheets %}
+      {% if debug_flag %}<link rel="stylesheet" href="{% static "_css/tailwind.compiled.css" %}">{% endif %}
```
Comment (KalobTaulien) [inline network-api/networkapi/templates/pages/base.html:34]:
```suggestion
      {% if debug %}<link rel="stylesheet" href="{% static "_css/tailwind.compiled.css" %}">{% endif %}
```

**Review by KalobTaulien — CHANGES_REQUESTED** [review]
This is looking good. Let's use a standardized "debug" term for our context processors so that it matches the backend. Everything else looks good. Ping me for a re-review when it's ready!

**Review by sabrinang — APPROVED** [review]
Great thanks for updating them to whole numbers for the naming Simon!

**Pull request merged**